_includes/master/manifests/calico-typha.yaml

```diff
@@ -90,19 +90,17 @@ spec:
           #- name: TYPHA_PROMETHEUSMETRICSPORT
           #  value: "9093"
         livenessProbe:
-          exec:
-            command:
-            - calico-typha
-            - check
-            - liveness
+          httpGet:
+            path: /liveness
+            port: 5473
```

Comment:
`5473` is Typha's main port but the healthcheck is on port `9098`

Reply:
Ack